code-examples/observer-demo/src/EconomistObserverDemo.java

```diff
@@ -0,0 +1,107 @@
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Set;
+
+public class EconomistObserverDemo {
+    public static void main(String[] args) {
+        Economist economist = new Economist();
+        Subscriber onlineSubscriber01 = new OnlineSubscription(economist);
```

Comment:
Can we put these in an array right away, and just iterate over them instead of doing the index thing down below?